From 19b307b15fb4dcc8f4c822dc4d0135bd52e5d297 Mon Sep 17 00:00:00 2001
From: Hariprasad Kelam <hkelam@marvell.com>
Date: Fri, 15 May 2020 14:20:10 +0530
Subject: [PATCH 0502/1921] octeontx2-af: Increase mbox response timeout to 3
 seconds

Upon receiving mbox request from PF to configure speed, fec, higig
AF sends commands to firmware and waits for max response time
which is 2.2 seconds.

In few scenarios firmware is taking more time to process the commands
and PF is not getting mbox reply as firmware command takes more time
than mbox response timeout which is 2 seconds. So increase the same
to 3 seconds to accommodate above case.

Change-Id: Ia114f01ae752d32da00f63cdf428aa4549da19f0
Signed-off-by: Hariprasad Kelam <hkelam@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/28645
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/mbox.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/mbox.h b/drivers/net/ethernet/marvell/octeontx2/af/mbox.h
index a42d54ee2a8d..c5ada30ee580 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/mbox.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/mbox.h
@@ -36,7 +36,7 @@
 
 #define INTR_MASK(pfvfs) ((pfvfs < 64) ? (BIT_ULL(pfvfs) - 1) : (~0ull))
 
-#define MBOX_RSP_TIMEOUT	2000 /* Time(ms) to wait for mbox response */
+#define MBOX_RSP_TIMEOUT	3000 /* Time(ms) to wait for mbox response */
 
 #define MBOX_MSG_ALIGN		16  /* Align mbox msg start to 16bytes */
 
-- 
2.31.1

